packages/firebase_crashlytics/firebase_crashlytics_platform_interface/lib/src/method_channel/method_channel_crashlytics.dart

```diff
@@ -87,9 +91,34 @@ class MethodChannelFirebaseCrashlytics extends FirebaseCrashlyticsPlatform {
     required String exception,
     required String information,
     required String reason,
+    bool fatal = false,
     List<Map<String, String>>? stackTraceElements,
   }) async {
     try {
+      /// "fatal" is an optional parameter that we're using to signal to the Crashlytic's service that this particular
+      /// error was a fatal one.  The below if statement is Firebase's prescribed method of signalling a fatal error.
+      if (fatal) {
+        try {
+          num currentUnixTimeSeconds =
+              (DateTime.now().millisecondsSinceEpoch / 1000).ceil();
+
+          await setCustomKey(_FATAL_FLAG, '$currentUnixTimeSeconds');
+
+          // TODO: once confirmation on the event name is received, reinstate analytics.logEvent below.
+          // await _analyticsChannel.invokeMethod('logEvent', <String, dynamic>{
+          //   'name': '_ae',
+          //   'parameters': {
+          //     'fatal': 1,
+          //     'timestamp': '$currentUnixTimeSeconds',
+          //   },
+          // });
+        } on MissingPluginException catch (e) {
```

Comment:
```suggestion
        } on MissingPluginException {
```

`e` us unused so can remove catch